spec/GrumPHP/Console/Helper/PathsHelperSpec.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace spec\GrumPHP\Console\Helper;
+
+use GrumPHP\Configuration\GrumPHP;
+use Symfony\Component\Filesystem\Filesystem;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+class PathsHelperSpec extends ObjectBehavior
+{
+    function let(GrumPHP $config, Filesystem $fileSystem)
+    {
+        $this->beConstructedWith($config, $fileSystem);
+    }
+
+    function it_is_initializable()
```

Comment:
You can remove the entire `it_is_initializable()` example, it's not very useful. :)